app/main/posts/modify/post-media.directive.js

```diff
@@ -16,39 +16,79 @@ function (
     return {
         restrict: 'E',
         replace: true,
-        require: '^^form',
+        require: 'ngModel',
         scope: {
-            mediaId: '=',
             media: '=',
```

Comment:
I'm actually not super happy we still have shared state with `media` but getting rid of it would require a big refactor